### PR TITLE
Documentation/note-about-key-generator-sequence-gaps

### PR DESCRIPTION
**Pull request by Simran-B**
Backport of #16007

### CODE REVIEW

File: Documentation/DocuBlocks/Rest/Collections/1_structs.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ determines how many copies of each shard are required to be
 in sync on the different DB-Servers. If there are less then these many copies
 in the cluster a shard will refuse to write. Writes to shards with enough
 up-to-date copies will succeed at the same time however. The value of
-*writeConcern* can not be larger than *replicationFactor*. _(cluster only)_
+`writeConcern` can not be larger than `replicationFactor`. _(cluster only)_
```
Comment (ansoboleva) [inline Documentation/DocuBlocks/Rest/Collections/1_structs.md:46]: can not --> cannot

File: Documentation/DocuBlocks/Rest/Collections/post_api_collection.md [hunk 2]
```diff
@@ -163,33 +168,33 @@ collection must contain the value of this attribute, followed by a colon,
...
-available at creation time and bail out otherwise. Set to *false* to disable
+Default is `true` which means the server will check if there are enough replicas
+available at creation time and bail out otherwise. Set to `false` to disable
 this extra check.
 
 @RESTRETURNCODES
 
 @RESTRETURNCODE{400}
-If the *collection-name* is missing, then a *HTTP 400* is
+If the `collection-name` is missing, then am *HTTP 400* is
```
Comment (ansoboleva) [inline Documentation/DocuBlocks/Rest/Collections/post_api_collection.md:193]: am --> an HTTP...

File: Documentation/DocuBlocks/Rest/Collections/post_api_collection.md [hunk 3]
```diff
@@ -163,33 +168,33 @@ collection must contain the value of this attribute, followed by a colon,
...
 returned.
 
 @RESTRETURNCODE{404}
-If the *collection-name* is unknown, then a *HTTP 404* is returned.
+If the `collection-name` is unknown, then am *HTTP 404* is returned.
```
Comment (ansoboleva) [inline Documentation/DocuBlocks/Rest/Collections/post_api_collection.md:197]: am --> an HTTP...

**Review by ansoboleva — APPROVED** [review]
please see the comments for a couple typos, otherwise LGTM

File: Documentation/DocuBlocks/Rest/Collections/post_api_collection.md (hunk 2 above)
Comment (Simran-B) [inline Documentation/DocuBlocks/Rest/Collections/post_api_collection.md:193]:
```suggestion
If the `collection-name` is missing, then an *HTTP 400* is
```

File: Documentation/DocuBlocks/Rest/Collections/post_api_collection.md (hunk 3 above)
Comment (Simran-B) [inline Documentation/DocuBlocks/Rest/Collections/post_api_collection.md:197]:
```suggestion
If the `collection-name` is unknown, then an *HTTP 404* is returned.
```

File: Documentation/DocuBlocks/Rest/Collections/1_structs.md (hunk 1 above)
Comment (Simran-B) [inline Documentation/DocuBlocks/Rest/Collections/1_structs.md:46]:
```suggestion
`writeConcern` cannot be larger than `replicationFactor`. _(cluster only)_
```

File: Documentation/DocuBlocks/Rest/Collections/post_api_collection.md [hunk 4]
```diff
@@ -116,10 +121,10 @@ It determines how many copies of each shard are required to be
 in sync on the different DB-Servers. If there are less then these many copies
 in the cluster a shard will refuse to write. Writes to shards with enough
 up-to-date copies will succeed at the same time however. The value of
-*writeConcern* can not be larger than *replicationFactor*. _(cluster only)_
+`writeConcern` can not be larger than `replicationFactor`. _(cluster only)_
```
Comment (Simran-B) [inline Documentation/DocuBlocks/Rest/Collections/post_api_collection.md:124]:
```suggestion
`writeConcern` cannot be larger than `replicationFactor`. _(cluster only)_
```

**Review by jsteemann — APPROVED** [review]
LGTM!

**Pull request merged**